src/app/portlets/dot-apps/dot-apps-import-export-dialog/dot-apps-import-export-dialog.component.html

```diff
@@ -0,0 +1,38 @@
+<dot-dialog
+    class="p-fluid"
+    *ngIf="showExportDialog"
+    [header]="dialogHeaderKey | dm"
+    [visible]="showExportDialog"
+    [actions]="dialogActions"
+    [appendToBody]="true"
+    (hide)="closeExportDialog()"
+>
+    <form [formGroup]="form" [ngSwitch]="action" novalidate>
+        <ng-container *ngSwitchCase="'Export'">
+            <input
+                dotAutofocus
+                autocomplete="new-password"
+                formControlName="password"
+                name="export-password"
+                pInputText
+                type="password"
+                [placeholder]="'apps.confirmation.export.password.label' | dm"
+            />
+        </ng-container>
+        <ng-container *ngSwitchCase="'Import'">
+            <input
+                dotAutofocus
+                autocomplete="new-password"
+                formControlName="password"
+                name="import-password"
+                pInputText
+                type="password"
+                [placeholder]="'apps.confirmation.import.password.label' | dm"
+            />
+            <p></p>
+            <input #importFile type="file" (change)="onFileChange($event.target.files)" />
+            <input type="hidden" name="fileHidden" formControlName="importFile"/> <!-- Validation Field -->
+        </ng-container>
+        <span>{{ errorMessage }}</span>
```

Comment:
Shouldn't this be `dot-field-validation-message`?

Reply:
that span is to only show errors that the BE throws after submitting the form, and looking at `dot-field-validation-message` it was made to show errors validating the form